app/src/main/java/com/github/h3lp3rs/h3lp/signIn/GoogleSignInAdaptor.kt

```diff
@@ -0,0 +1,90 @@
+package com.github.h3lp3rs.h3lp.signIn
+
+import android.app.Activity
+import android.content.Intent
+import android.widget.Toast
+import androidx.activity.result.ActivityResult
+import androidx.core.content.ContextCompat.startActivity
+import com.github.h3lp3rs.h3lp.MainPageActivity
+import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.common.api.ApiException
+import com.google.android.gms.tasks.Task
+import com.google.firebase.auth.AuthResult
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.GoogleAuthProvider
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+
```

Comment:
Very good documentation for the methods. I would however add a comment at the top of what this class is about.